packages/vite-plugin-qrcode/package.json

```
@@ -24,7 +24,7 @@
     "test": "tsm node_modules/uvu/bin.js tests"
   },
   "engines": {
-    "node": "^12.20 || ^14.13.1 || >= 16"
+    "node": ">=14"
```

this allows node 15 and 17, both of them are EOL

```suggestion
    "node": "^14.13.1 || ^16 || >=18"
```